source/locale/pt_BR/LC_MESSAGES/faq.po

```diff
@@ -249,667 +169,363 @@ msgstr "1 Atualize o repositório do *docker compose* com a última versão."
 
-#: ../../source/faq/faq.rst:448
-msgid ""
-"**Using flowbroker to pre-process data**: for more flexible ways of data "
-"manipulation, you can use flows. They can process/transform data so they "
-"can be properly sent to your application via HTTP request, or stored in a"
-" virtual device (which can be used to generate notifications as "
-"previously described)."
-msgstr ""
-"**Usando o flowbroker para pré-processar dados**: eles podem ajudar a "
-"processar e transformar os dados para que possam ser enviados "
-"corretamente para o seu aplicativo por meio de solicitação HTTP ou "
-"armazenados em um dispositivo virtual (que pode ser usado para gerar "
-"notificações conforme descrito anteriormente). "
+#: ../../source/faq/faq.rst:449
+msgid "**Using flowbroker to pre-process data**: for more flexible ways of data manipulation, you can use flows. They can process/transform data so they can be properly sent to your application via HTTP request, or stored in a virtual device (which can be used to generate notifications as previously described)."
+msgstr "**Usando o flowbroker para pré-processar dados**: eles podem ajudar a processar e transformar os dados para que possam ser enviados corretamente para o seu aplicativo por meio de solicitação HTTP ou armazenados em um dispositivo virtual (que pode ser usado para gerar notificações conforme descrito anteriormente)."
 
-#: ../../source/faq/faq.rst:455
-msgid ""
-"All these endpoints should bear an access token, see more :ref:`Getting "
-"access token`."
-msgstr ""
-"Todos esses *endpoints* devem ter um *token* de acesso. Para isso, "
-"confira :ref:`Obtendo um token de acesso`."
+#: ../../source/faq/faq.rst:456
+msgid "All these endpoints should bear an access token, see more :ref:`Getting access token`."
+msgstr "Todos esses *endpoints* devem ter um *token* de acesso. Para isso, confira :ref:`Obtendo um token de acesso`."
 
-#: ../../source/faq/faq.rst:457
-msgid ""
-"Check the documentation API for  *History* and *Kafka WS* in :doc"
-":`./../components-and-apis`. And to a tutorial on how to use the flow, "
-"check :doc:`../flow`."
-msgstr ""
-"Verifique a documentação da API do *History* e do *Kafka WS* em :doc"
-":`./../components-and-apis`.  E para um tutorial de como usar o fluxo "
-"veja :doc:`../flow`."
+#: ../../source/faq/faq.rst:458
+msgid "Check the documentation API for  *History* and *Kafka WS* in :doc:`./../components-and-apis`. And to a tutorial on how to use the flow, check :doc:`../flow`."
+msgstr "Verifique a documentação da API do *History* e do *Kafka WS* em :doc:`./../components-and-apis`.  E para um tutorial de como usar o fluxo veja :doc:`../flow`."
 
 #~ msgid "Check a tutorial using history in :ref:`Checking historical data`."
 #~ msgstr "Veja um tutorial usando o *history* :ref:`Checking historical data`."
 
 #~ msgid "It takes a role as an enabler platform with:"
-#~ msgstr "A dojot desempenha o papel de plataforma facilitadora com:"
+#~ msgstr ""
```

Comment:
Need add the translation here?

Reply:
This are old translation, they are commented, they can be removed.

Reply:
I removed these comments